packages/platform-redis/src/redis-interceptor.service.ts

```diff
@@ -0,0 +1,38 @@
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { PATTERN_METADATA } from '@nestjs/microservices/constants';
+import { AbstractInterceptorService } from '@ogma/nestjs-module';
+
+@Injectable()
+export class RedisParser extends AbstractInterceptorService {
+  getCallPoint(context: ExecutionContext): string {
+    return JSON.stringify(
+      this.reflector.get(PATTERN_METADATA, context.getHandler()),
+    );
+  }
+
+  getCallerIp(context: ExecutionContext): any {
+    const data = this.getData(context);
+    return data?.ip || '';
```

Comment:
This line did not cover.

```typescript
describe('getCallerIp', () => {
  it('should return an empty string', () => {
    const ctxMock = createMock<ExecutionContext>({
      switchToRpc: () => ({
        getData: () => ({}),
      }),
    });
    expect(parser.getCallerIp(ctxMock)).toBe('');
  });
  it('should return the ip', () => {
    const ctxMock = createMock<ExecutionContext>({
      switchToRpc: () => ({
        getData: () => ({
          ip: '127.0.0.1',
        }),
      }),
    });
    expect(parser.getCallerIp(ctxMock)).toBe('127.0.0.1');
  });
});
```
Any ideas?

Reply:
Uh, yeah, you can add in one more test like so:
```ts
it('should return an empty string', () => {
  const ctxMock = createMock<ExecutionContext>({
    switchToRpc: () => ({
      getData: () => undefined,
    })
  });
  expect(parser.getCallerIp(ctxMock)).toBe('');
});
```
For now, it's fine without this, it's a case that would practically never happen.

Reply:
Should I add one?

Reply:
Your call. I'm not too worried about it